src/app/debug/table/table-settings-modal/table-settings-modal.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-template #modal let-modal>
   <div class="modal-header">
     <h5 class="modal-title" id="settingsModal">Options</h5>
-    <button id="debugSettingsModalClose" data-cy-close-settings-modal type="button" class="close"
+    <button id="debugSettingsModalClose" data-cy-settings-modal="close" type="button" class="close"
```

Comment:
Modal mag je hier weglaten
```suggestion
    <button id="debugSettingsModalClose" data-cy-settings="close" type="button" class="close"
```